packages/next/src/bin/next.ts

```diff
@@ -264,9 +264,11 @@ program
     '--no-inline-config',
     'Prevents comments from changing config or rules.'
   )
-  .option(
-    '--report-unused-disable-directives',
```

Comment:
Are we completely removing this option? Based on the docs it still seems to be a valid option?

Reply:
@coltonehrman Yes, we're removing this. I'll update the <picture data-single-emoji=":docs:" title=":docs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/docs/172e1b9b7d137b8a.png" alt=":docs:" align="absmiddle"></picture> as well!